lightning/src/ln/functional_tests.rs

```diff
@@ -2965,11 +2965,7 @@ fn test_htlc_on_chain_timeout() {
 
 	let chanmon_cfgs = create_chanmon_cfgs(3);
 	let node_cfgs = create_node_cfgs(3, &chanmon_cfgs);
-	let mut cfg = UserConfig::default();
-	// This test was written when the CLTV_EXPIRY_DELTA was fixed at 12*6, and expects it.
-	cfg.channel_options.cltv_expiry_delta = 12*6;
-	cfg.channel_options.announced_channel = true;
-	let node_chanmgrs = create_node_chanmgrs(3, &node_cfgs, &[Some(cfg.clone()), Some(cfg.clone()), Some(cfg.clone())]);
+	let node_chanmgrs = create_node_chanmgrs(3, &node_cfgs, &[None, None, None]);
```

Comment:
Unclear what 444c154d6e660ad4228d0ea7b04ce72d7a2c1c01 is a fixup for.

Reply:
It reverts a hunk from #849 - I could probably not revert it, but I'd need to redo the test changes here to figure out how it used to be.